command_line.go

```diff
@@ -26,6 +26,7 @@ func ParseCommandLine(app *kingpin.Application) (*cmd.CommandLine, error) {
 	// 2. Parse flags
 	cmdLine := cmd.CommandLine{}
 	app.Flag("config", "path to rsc config file").Short('c').Default(path.Join(os.Getenv("HOME"), ".rsc")).StringVar(&cmdLine.ConfigPath)
+	app.Flag("retry", "number that rsc would retry on non-successful API response").Short('R').Default("0").IntVar(&cmdLine.Retry)
```

Comment:
`-t` is also already used for `--timeout` :) So we can stick with `-R` or just drop short option as we do for some options